docroot/modules/humsci/hs_courses/modules/hs_courses_importer/hs_courses_importer.module

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @file
+ * hs_coures_importer.module
+ */
+
+use Drupal\search_api\IndexInterface;
+use Drupal\node\NodeInterface;
+
+/**
+ * Implements hook_search_api_index_items_alter().
+ *
+ * Our course importer imports multiple nodes for each course. Its possible that
+ * a single course could have dozens of sections and therefore dozens of nodes.
+ * Since the data is identical for each section except for date and times, we
+ * can exclude all sections except 1 from indexing. This will prevent a single
+ * course from consuming the search results if a user is looking for a
+ * non-course content, but it has matching results as a course.
+ */
+function hs_courses_importer_search_api_index_items_alter(IndexInterface $index, array &$items) {
```

Comment:
I like this approach of not indexing nodes better than filtering them out at run time but if date does matter this might not be a viable approach.

Reply:
since we're using rabbit hole to bounce the user off site to explorecourses.stanford.edu the only issue with this would be if a user explicitly searched for `december` or something like that. if in the future H&S want's to have the user stay on site and land on the course, we might have to revisit this.

Reply:
Gotcha, thanks.